test/module/irohad/ametsuchi/ametsuchi_mocks.hpp

```diff
@@ -167,7 +167,7 @@ namespace iroha {
     expected::Result<std::unique_ptr<MutableStorage>, std::string>
     createMockMutableStorage() {
       return expected::makeValue<std::unique_ptr<MutableStorage>>(
-          std::move(std::make_unique<MockMutableStorage>()));
+          std::make_unique<MockMutableStorage>());
```

Comment:
What is the point to remove `std::move`?

Reply:
The parameter passed is already a rvalue

Reply:
https://github.com/hyperledger/iroha/pull/963#discussion_r167930573